error[E0596]: cannot borrow `ring` as mutable, as it is not declared as mutable
  --> tests/compile-fail/fixed_file_use_after_unregister.rs:21:19
   |
21 |     let _future = ring.read_fixed(&fixed_stdin, buffer.as_mut_slice()); //~ ERROR: potential use after unregister
   |                   ^^^^ cannot borrow as mutable
   |
help: consider changing this to be mutable
   |
17 |     let mut ring = Ring::new(32).unwrap();
   |         +++

error[E0596]: cannot borrow `buffer` as mutable, as it is not declared as mutable
  --> tests/compile-fail/fixed_file_use_after_unregister.rs:21:49
   |
21 |     let _future = ring.read_fixed(&fixed_stdin, buffer.as_mut_slice()); //~ ERROR: potential use after unregister
   |                                                 ^^^^^^ cannot borrow as mutable
   |
help: consider changing this to be mutable
   |
18 |     let mut buffer = safer_ring::PinnedBuffer::with_capacity(1024);
   |         +++
